Popup/tests/Popup-specs.js

```diff
@@ -30,36 +44,82 @@ describe('Popup specs', () => {
 		expect(actual).toBe(expected);
 	});
 
-	test('should be rendered with centered content if centered is set to true', () => {
-		const popup = mount(
-			<Popup centered />
-		);
+	describe('with centered content', function () {
+		test('should be rendered with centered content if centered is set to true', () => {
+			const popup = mount(
+				<Popup centered />
+			);
 
-		const expected = true;
-		const actual = popup.prop('centered');
+			const expected = true;
+			const actual = popup.prop('centered');
 
-		expect(actual).toBe(expected);
+			expect(actual).toBe(expected);
+		});
+
+		test('should not be rendered with centered content if centered is set to false', () => {
+			const popup = mount(
+				<Popup centered={false} />
+			);
+
+			const expected = false;
+			const actual = popup.prop('centered');
+
+			expect(actual).toBe(expected);
+		});
 	});
 
-	test('should not be rendered with centered content if centered is set to false', () => {
-		const popup = mount(
-			<Popup centered={false} />
-		);
+	describe('with position center', function () {
+		test('should have position=center when no position is specified', () => {
+			const popup = mount(
+				<PopupBase />
+			);
 
-		const expected = false;
-		const actual = popup.prop('centered');
+			const expected = 'center';
+			const actual = popup.prop('position');
 
-		expect(actual).toBe(expected);
+			expect(actual).toBe(expected);
+		});
+
+		test('should not have center class', () => {
```

Comment:
What about updating this to explain what is the key point here? If I understand correctly, this test case is to distinguish `position` prop's value 'center' and class 'center' by `centered` prop. Am I right? :)

Reply:
I did not explain it correctly, true. What I wanted to point out was that when it has position=center it has default styling, without adding an extra "center" class as it does when it has position=top.

Reply:
I think maybe we could just skip this test.

Reply:
I will delete this test for now and add it in this task: PLAT-129752: Sync Popup positioning with Sandstone.